CMakeLists.txt

```diff
@@ -5,6 +5,8 @@ set(CMAKE_CXX_STANDARD 11)
 
 list(APPEND CMAKE_FIND_ROOT_PATH ${CMAKE_BINARY_DIR}/third_party)
 
+cmake_policy(SET CMP0025 NEW)
```

Comment:
I can do it if you insist, but I tend to disagree on that :-). "CMP0025" is so unique that you cannot miss the [CMake documentation](https://cmake.org/cmake/help/v3.0/policy/CMP0025.html) when Googling it, and its first line says:

> https://cmake.org/cmake/help/v3.0/policy/CMP0025.html

I generally don't like adding comments, but I agree it is necessary sometimes when we have to do weird tricks. This is not a trick at all. To me it would be the same as as putting a comment above `std::shared_ptr<...>` to explain what a shared pointer is. It is general knowledge, and I believe my source code is not the place for general documentation :D. Comments tend to get outdated and clutter the code.

Again, if you insist, I can add a comment there :D.

Reply:
Ok, agreed.